### DISCUSSION TITLE
Private and Public Star Lists

### DISCUSSION THREAD

**Discussion by httpanand** [community/community#8608] [Other Feature Feedback, Questions, & Ideas] [answered]
Please add a feature to make Star lists public and private ..  It'll be helpful to some users like me . Thank You

**Comment by jecastrom** [comment]
I agree. The list feature is fantastic. However, I notice the list URL is not public, even being the repo's public.  It would be beneficial to have their repos organised and share them, for example, with employers, only a specific list of projects, rather than the profile URL full of projects and people getting lost finding stuff.

**Comment by glewe** [comment]
+1

**Comment by cliffordfajardo** [comment]
I also hit the 16 limit limitation quickly. 
my use case:
- I typically view my GitHub newsfeed feed and GitHub trending list several times a week and I like to categorize repos i want to experiment with

Before GitHub lists, I created my own lists using an external store (notion database) but the need to context switch out of the GitHub web app  to save the repo and categorize it  usually caused me to just defer the action 😢 due to the extra step / time needed specially since I use GitHub on mobile a lot (I use it to catch up on updates / view GitHub feed like I do my twitter feed )


Related
- https://github.com/github/feedback/discussions/8633
- https://github.com/github/feedback/discussions/9672#discussion-3781923
- https://github.com/github/feedback/discussions/9513

**Reply by melMass** [reply]
+1 I had the exact same workflow (notion based) and this feature is making it much better but I hit the limit pretty quickly 😢

**Comment by fluteds** [comment]
It would be nice if private lists worked like private Gists in that anyone with the link can view it.

**Reply by ZelnickB** [reply]
It might also be nice to add an option also for private lists like private repositories (you can invite people to view and/or edit the lists)

**Comment by junibrosas** [comment]
Came here after reaching the 16 limitation. Github's list feature is awesome and very helpful and I slowly migrated my starred repos from an external manager.

**Comment by ghost** [comment]
`+ 1`

**Comment by telemakhos** [comment]
Highly demanded feature and simple to implement. Why is taking so long?

**Reply by httpanand** [reply]
yeah it'll be very useful for many i guess !

**Reply by svengreb** [reply]
Sounds like you must be a pure genius and can implement it for “them“ in some days, can‘t you?
Putting the sarcasm aside, please don‘t demand for things without any knowledge about the high complexity of gigantic systems like GitHub or any other kind of global-serving platforms. Changing a single line of code in such systems can have fatal consequences.
Please stop to beg for something and produce non-productive noise in such large threads instead of contributing useful information.

**Reply by telemakhos** [reply]
Sounds like you're a victim of microservices... 😂 https://www.youtube.com/watch?v=y8OnoxKotPQ

**Comment by ooahz** [comment]
+1

**Comment by Hexx333** [comment]
I agree.

**Comment by ZelnickB** [comment]
I support increasing (or removing, if that isn't too much to ask) the 16-list limit *and* allowing lists to be made private.

**Comment by Hexx333** [comment]
Ok

pt., 2 cze 2023 o 22:17 Ben Zelnick ***@***.***> napisał(a):

**Comment by Hexx333** [comment]
Thank you so much for the great news!

sob., 3 cze 2023, 18:28 użytkownik Mtee74 ***@***.***>
napisał:

**Comment by Titlap77** [comment]
> Please add a feature to make Star lists public and private ..  It'll be helpful to some users like me . Thank You

**Comment by dotrinh-USE-Inc** [comment]
same here

**Comment by LiyJessica** [comment]
+1

**Comment by abtin67** [comment]
Hello Please someone help me how to see and use the source code

**Comment by arvinmoj** [comment]
I understand your request, but I have to advise against making star lists public or private. 

making star lists private could create a sense of exclusivity, elitism, or secrecy, which may not align with the values of inclusivity, transparency, and openness that are essential for maintaining a safe and respectful community.

**Reply by svengreb** [reply]
What does all of this have to do with privacy settings for a single user? These topics you listed have absolutely nothing to do with the topic in this discussion.
This is like saying that you have to make your bank account balance public, otherwise other customers of the bank might feel excluded.
The projects you like are your own opinion and when you don‘t want to share them this should be your own decision.
In fact all your listed points are the absolute opposite of whats important: protecting the privacy of users instead of making them the perfect “glass figure“ everyone can suck information for data collection from.

Allowing to make private lists fits perfectly into features like creating private repositories for, well, the code that is your property.

**Comment by yorickshan** [comment]
same here

**Comment by LiteBrite82** [comment, marked as answer]
Hi All!

This discussion is being locked and linked to a single discussion to better track user feedback: Dec 9, 2021 [Private List #8626](https://github.com/orgs/community/discussions/8626) Please continue any further discussion there. 

All data and feedback in this discussion is still being tracked and will be combined (internally) with the above mentioned discussion.

**Reply by LiteBrite82** [reply]
💬 Your Product Feedback Has Been Submitted 🎉

Thank you for taking the time to share your insights with us! Your feedback is invaluable as we build a better GitHub experience for all our users.

Here's what you can expect moving forward ⏩

Your input will be carefully reviewed and cataloged by members of our product teams. 
Due to the high volume of submissions, we may not always be able to provide individual responses.
Rest assured, your feedback will help chart our course for product improvements.
Other users may engage with your post, sharing their own perspectives or experiences. 
GitHub staff may reach out for further clarification or insight. 
We may 'Answer' your discussion if there is a current solution, workaround, or roadmap/changelog post related to the feedback.
Where to look to see what's shipping 👀

Read the [Changelog](https://github.blog/changelog/) for real-time updates on the latest GitHub features, enhancements, and calls for feedback.
Explore our [Product Roadmap](https://github.com/orgs/github/projects/4247), which details upcoming major releases and initiatives.
What you can do in the meantime 💻

Upvote and comment on other user feedback Discussions that resonate with you.
Add more information at any point! Useful details include: use cases, relevant labels, desired outcomes, and any accompanying screenshots.
As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities.

Thank you once again for your contribution to making GitHub even better! We're grateful for your ongoing support and collaboration in shaping the future of our platform. ⭐
